### PR TITLE
Restrict continuous-integration/travis-ci/push to only the master branch

### PR DESCRIPTION
**Pull request by rnelson0**

### CODE REVIEW

**Comment by nanliu** [comment]
IMO, we should only have the master branch in this repo. Any pending work should be done on contributor fork and not the main repo.

**Comment by ffrank** [comment]
Will Travis still consider PRs for testing?

**Comment by rnelson0** [comment]
Yes, the task `continuous-integration/travis-ci/pr` is not affected in any way.

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ sudo: false
 language: ruby
 cache: bundler
 bundler_args: --without system_tests
+branches:
+  only:
+    - master
```
Comment (igalic) [inline .travis.yml]: this should go to either .sync.yml, or to modulesync_config

Comment (jyaworski) [inline .travis.yml]: I can help with this.

**Pull request merged**